package.json

```diff
@@ -155,6 +159,7 @@
     "node-sass": "^4.5.3",
     "null-loader": "^0.1.1",
     "polyfill-function-prototype-bind": "0.0.1",
+    "prettier": "^1.14.3",
```

Comment:
Could you please also include the yarn.lock updates?

Reply:
I'm not sure why the yarn.lock didn't get updated. Weird. Will look into that.